### ISSUE TITLE
Cannot see number genie nodejs working on simulator.

### ISSUE THREAD

**Issue by cbothra** [actions-on-google/dialogflow-number-genie-nodejs#10]
I followed all the steps mentioned in readme file and successfully able to import the actions and deploy the node on firebase. When I goto the simulator to check then I get default response and get the following error on Google actions simulator:

` "message": "Webhook error (206)"`

On DialogFlow simulator, in the diagnostic report I see following error in the fulfillment response:

```
{
  "error": "Converting circular structure to JSON"
}
```

**Comment by cbothra** [comment]
Following is the log from google cloud console:
`
TypeError: Converting circular structure to JSON

        at Object.stringify (native)
        at Object.exports.clone (common.js:33)
        at Object.exports.stringify (common.js:35)
        at Function.<anonymous> (/user_code/node_modules/actions-on-google/dist/service/dialogflow/dialogflow.js:127)
        at next (native)
        at (/user_code/node_modules/actions-on-google/dist/service/dialogflow/dialogflow.js:22)
        at __awaiter (/user_code/node_modules/actions-on-google/dist/service/dialogflow/dialogflow.js:18)
        at Function.handler (/user_code/node_modules/actions-on-google/dist/service/dialogflow/dialogflow.js:84)
        at Object.<anonymous> (/user_code/node_modules/actions-on-google/dist/assistant.js:55)
        at next (native)

`

**Comment by Canain** [comment]
Hey, thanks for reporting this!

It's a bug with the most recent client library release. We are working on a fix right now.

**Comment by Canain** [comment]
In the meanwhile, you can manually downgrade to a previous version:

```
cd functions
npm install --save actions-on-google@2.0.1
```

**Comment by ajoevarghese** [comment]
Hey, I am also facing the same issue, my app when I am deploying to cloud functions, using firebase deploy. The deployed code seems to be throwing this error from GCF. 
However the app is working fine locally, using firebase serve.
Even after manually downgrading to 2.0.1 and even 2.0.0. still the same issue persists, when i do a firebase deploy

**Comment by Canain** [comment]
@ajoevarghese did you save the version in `package.json`?

I edited the command so that npm also saves the new version. It should be

```
cd functions
npm install --save actions-on-google@2.0.1
```

**Comment by ajoevarghese** [comment]
@Canain, Yep, I have already tried that, its still not working. 
In the package.json, 
`"dependencies": {
    "actions-on-google": "^2.0.1",`
the dependency I have updated and tried on 2.0.1 and 2.0.0.

**Comment by Canain** [comment]
The dependency you have listed is actually fuzzy matching for `>=2.0.1` and `<3.0.0`.
So it actually matches `2.1.1` and `2.1.0` still.

You need to remove the `^` symbol so it says

`"actions-on-google": "2.0.1",`

**Comment by ajoevarghese** [comment]
@Canain It is working when i changed it to
`"actions-on-google": "2.0.1",
`
Thanks a lot for the help.

**Comment by Canain** [comment]
Should be fixed now with `2.1.2`, delete the installed packages with `rm -rf node_modules package-lock.json` and then do `npm install` again
